src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -77,7 +78,7 @@ def split_tensor_along_last_dim(tensor, num_partitions, contiguous_split_chunks=
     return tensor_list
 
 
-def attention_mask_func(attention_scores, attention_mask, causal_mask):
+def attention_mask_func(attention_scores, attention_mask, causal_mask, prefix_mask):
```

Comment:
Since prefix_mask is actually a variant of causal mask, maybe we don't need `prefix_mask` and instead put it directly to the `causal_mask` argument